src/marine/godae/profile2ioda.py

```diff
@@ -158,7 +158,7 @@ def _to_nlocs(self):
             dtg = datetime.strptime(self.odata['ob_dtg'][n], '%Y%m%d%H%M')
 
             # Diff. between ob. time and time in the file (hrs)!
-            ddtg = (dtg - self.date).seconds/3600.
+            ddtg = (dtg - self.date).total_seconds/3600.
```

Comment:
**total_seconds** is a function, needs **()**
same with other 2 locations

Reply:
thanks for the catch.